schema/project-level/codelists/documentType.csv

```diff
@@ -93,3 +93,4 @@ consultationMeetingInvitation,Consultation meeting invitation,A notice about an
 materialTestResults,Material test results,Documentation of results of tests conducted on construction materials.,OC4IDS
 climateMeasures,Climate measures,Documentation of the measures adopted by the project to adapt to or mitigate against climate change.,OC4IDS
 maintenancePlan,Maintenance plan,Documentation that describes work to prevent the breakdown or malfunctioning of an asset.,OC4IDS
+environmentalMeasures,Environmental measures,Documentation of the measures adopted by the project to mitigate or remedy environmental impacts.,OC4IDS
```

Comment:
Move this and climateMeasures under conservationMeasures (and order them alphabetically).